test/Confluent.SchemaRegistry.Serdes.UnitTests/Configuration.cs

```diff
@@ -33,7 +33,7 @@ public ConfigurationTests()
         {
             testTopic = "topic";
             var schemaRegistryMock = new Mock<ISchemaRegistryClient>();
-            schemaRegistryMock.Setup(x => x.RegisterSchemaAsync(testTopic + "-value", It.IsAny<string>())).ReturnsAsync(
+            schemaRegistryMock.Setup(x => x.RegisterSchemaAsync(testTopic + "-value", It.IsAny<Schema>())).ReturnsAsync(
```

Comment:
This one is different from the other changes in this PR.

Reply:
i missed it in the SR PR.